estate-services/src/main/java/org/egov/ps/model/Application.java

```diff
@@ -202,4 +205,32 @@ public Application addApplicationDocumentsItem(Document applicationDocumentItem)
 	public String getMDMSModuleName() {
 		return String.format("%s_%s_%s", this.getBranchType(), this.getModuleType(), this.getApplicationType());
 	}
+	
+	/**
+	 * Pending consumer code. This needs to be saved in the database for online
+	 * payments.
+	 */
+	@JsonProperty("paymentConsumerCode")
+	@Size(max = 256, message = "payment consumer code must be between 0 and 256 characters in length")
+	private String paymentConsumerCode;
```

Comment:
There should be no new paymentConsumerCode for an application. The application number should be used to generated demand, bill etc..